backend/src/org/commcare/xml/EntryParser.java

```diff
@@ -149,6 +153,18 @@ private void parseInstance(Hashtable<String, DataInstance> instances) {
 
     private SyncPost parsePost() throws InvalidStructureException, IOException, XmlPullParserException {
         String url = parser.getAttributeValue(null, "url");
+
```

Comment:
not in this PR, but line 136 above is the only one still using `this.parser` instead of `parser`